event.go

```diff
@@ -0,0 +1,376 @@
+}
+
+// BridgeBlindTransfer events signify that a blind transfer has occurred
+type BridgeBlindTransfer struct {
+	EventData
+	Bridge Bridge `json:"bridge,omitempty"`
+	ChannelData
+	Context        string  `json:"context"`
+	Extension      string  `json:"exten"`
+	External       bool    `json:"external"`
+	ReplaceChannel Channel `json:"external,omitempty"`
+	Result         string  `json:"result"`
+	Transferee     Channel `json:"transferee,omitempty"`
+}
+
+// BridgeCreated events indicate a bridge has been created
+type BridgeCreated struct {
+	EventData
+	Bridge Bridge `json:"bridge"`
+}
+
+// BridgeDestroyed events indicate a bridge has been destroyed
+type BridgeDestroyed struct {
+	EventData
+	Bridge Bridge `json:"bridge"`
+}
+
+// BridgeMerged events indicate a bridge has been merged into another (bridge)
+type BridgeMerged struct {
+	EventData
+	Bridge      Bridge `json:"bridge"`      // New bridge
+	Bridge_from Bridge `json:"bridge_from"` // Old (independant) bridge -- TODO: verify this assumption
+}
+
+/* --------- Channel Events ---------------- */
+
+// ChannelCallerId events indicate a channel's caller Id information has changed
+type ChannelCallerId struct {
+	EventData
+	Caller_presentation     int    `json:"caller_presentation"`     // The numeric portion
+	Caller_presentation_txt string `json:"caller_presentation_txt"` // The textual portion
+	ChannelData
+}
+
+// ChannelConnectedLine events indicate a channel has changed its connected line
+type ChannelConnectedLine struct {
+	EventData
+	ChannelData
+}
+
+// ChannelCreated events indicate a channel has been created
+type ChannelCreated struct {
+	EventData
+	ChannelData
+}
+
+// ChannelDestroyed events indicate a channel has been destroyed
+type ChannelDestroyed struct {
+	EventData
+	Cause     int    `json:"cause"`
+	Cause_txt string `json:"cause_txt"`
+	ChannelData
+}
+
+// ChannelDialplan events indicate a channel has changed its location in the dialplan
+//   NOTE: This event also most likely implies the channel is leaving the control of this
+//   ARI application
+type ChannelDialplan struct {
+	EventData
+	ChannelData
+	Dialplan_app      string `json:"dialplan_app"`      // The application which is to be executed
+	Dialplan_app_data string `json:"dialplan_app_data"` // The data to be passed to the application
+}
+
+// ChannelDtmfReceived events indicate a channel has received a DTMF tone.
+//   NOTE: this event is sent at the _end_ of the DTMF tone (and there is no indication
+//   for the _start_ of the DTMF tone)
+type ChannelDtmfReceived struct {
+	EventData
+	ChannelData
+	Digit       string `json:"digit"`       // The DTMF digit which was received (0-9A-E*#)
+	Duration_ms int    `json:"duration_ms"` // The duration of the DTMF tone, in milliseconds
+}
+
+// ChannelEngteredBridge events indicate that a channel has joined a bridge
+type ChannelEnteredBridge struct {
+	EventData
+	Bridge Bridge `json:"bridge"`
```

Comment:
Done in https://github.com/CyCoreSystems/ari/pull/28/commits/6067360c2151c6dfad49c214e58a3bd54b95977d